### PR TITLE
mpfs: Initial commit to add GPIO mux configuration

### PR DESCRIPTION
**Pull request by jlaitine**
*not* much tested yet

### CODE REVIEW

File: arch/risc-v/src/mpfs/hardware/mpfs250t_484_pinmap.h [hunk 1]
```diff
@@ -0,0 +1,136 @@
+/****************************************************************************
+ * arch/risc-v/src/mpfs/hardware/mpfs250t_pinmap.h
```
Comment (jpaali) [inline arch/risc-v/src/mpfs/hardware/mpfs250t_484_pinmap.h:2]: mpfs250t_484_pinmap.h

File: arch/risc-v/src/mpfs/hardware/mpfs250t_484_pinmap.h [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                                             MSSIO_MUX_BANK1_REG_OFFSET(pin))
+#define MSSIO_MUX_BANK_REG(bank,pin) (MPFS_SYSREG_BASE +                       \
+                                      MSSIO_MUX_BANK_REG_OFFSET(bank,pin))
+
+/* Default EC configuration for all GPIOS */
+#define MSSIO_EC_DEFAULT (0x0428 << GPIO_EC_SHIFT)
+
+/* Basic GPIO definitions for MSSIO */
+
+
```
Comment (jpaali) [inline arch/risc-v/src/mpfs/hardware/mpfs250t_484_pinmap.h:87]: check failure about too many blank lines

**Review by jpaali — CHANGES_REQUESTED** [review]
looks good

**Review by jpaali — APPROVED** [review]

**Comment by jlaitine** [comment]
Still added defconfigs

**Pull request merged**